src/domain/app/i18n/en.json

```
@@ -1831,50 +1831,67 @@
       "title": "Are you sure you want to cancel the registration?"
     },
     "form": {
-      "buttonDeleteSignup": "Delete participant",
       "buttonSave": "Save participant",
       "buttonUpdateParticipantAmount": "Update",
+      "contactPerson": {
+        "cannotEditContactPersonNotification": {
+          "linkText": "Edit the signup group",
+          "text": "The signup group's contact person information cannot be edited from this page. Contact information can be edited from the editing page of the signup group.",
```

```suggestion
          "text": "The signup group's contact person information cannot be edited from this page. Contact information can be edited from the signup group's editing page.",
```
This phrase is a little bit shorter and since "group" is referring to people, the s-genitive is commonly used in that case.